content/common/language-guide-overview.md

```diff
@@ -0,0 +1,533 @@
+title = "Language Guide Overview"
+template = "common_main"
+date = "2022-01-01T00:00:01Z"
+enable_shortcodes = true
+[extra]
+
+---
+
+The following code block outlines, in the context of the Spin framework, what languages have SDK support, and what SDK features are available in each of those languages:
+
+{{ tabs "sdk-type" }}
+
+{{ startTab "C#/.NET"}}
```

Comment:
Our other `sdk-types` tab sets have the order and terminology: Rust | TypeScript | Python | TinyGo

We should keep the names consistent (I don't care about JavaScript vs TypeScript, or Go vs TinyGo, as long as it's consistent), so that a user's choice of language travels with them.

I would also suggest keeping the order consistent both for looking organised, and for muscle memory.  Languages that aren't yet everywhere, such as C#, would go at the end.

Aside: at present, I would _never_ put C# first, _anywhere._   The SDK is not yet mature enough to be placed in the most prominent position.

Reply:
This is great feedback @itowlson, thank you!

**Order**
I initially just wrote the page out in alphabetical order, on purpose, so there was no bias towards any language. I am hearing you, and it does make sense to put more mature offerings at eye level; for sure. So essentially duplicating the following visual that already exists elsewhere (and then adding any new languages at the end (or on the right-hand side).

<img width="530" alt="Screenshot 2023-03-15 at 8 29 55 am" src="https://user-images.githubusercontent.com/9831342/225156845-5d7c9642-27e1-4141-8dee-5222ebff09d1.png">

**Consistency**
Thanks for the `sdk-types` consistency comment also; I 100% agree with this, just wasn't sure which way to go but I guess the existing `sdk-types` in existing code blocks are already setting the standard so it makes sense to match these with what already exists. If we change our minds (for whatever reason; then we reflect those changes throughout the whole developer docs for consistency).